packages/aws-cdk-lib/aws-cloudwatch-actions/README.md

```diff
@@ -40,4 +40,16 @@ alarm.addAlarmAction(
 );
 ```
 
+## Lambda Action Example
+
+```ts
+import * as lambda from 'aws-cdk-lib/aws-lambda'
+declare const alarm: cloudwatch.Alarm;
+declare const fn: lambda.Function;
+
```

Comment:
Could you add a description here for what is being created similar to what's done in the other examples?